src/ucs/debug/instrument.h

```diff
@@ -13,8 +13,17 @@
 #endif
 
 #include <ucs/sys/preprocessor.h>
+#include <ucs/datastruct/list.h>
 #include <ucs/time/time.h>
 
+typedef enum ucs_instrumentation_types {
+    UCS_INSTRUMENT_TYPE_TX = 0,
+    UCS_INSTRUMENT_TYPE_RX,
+    UCS_INSTRUMENT_TYPE_IB_TX,
```

Comment:
There is no "ib" in test_instr.cc ... it is in instrument.h because IB has it's own record. would you like to change the name to something like "UCS_INSTRUMENT_TYPE_WR_TX" ?